planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/util.cpp

```diff
@@ -719,8 +719,17 @@ lanelet::Lanelet createDepartureCheckLanelet(
   };
 
   const auto getMostInnerLane = [&](const lanelet::ConstLanelet & lane) -> lanelet::ConstLanelet {
-    return left_side_parking ? route_handler.getMostRightLanelet(lane, false, true)
-                             : route_handler.getMostLeftLanelet(lane, false, true);
+    if (left_side_parking) {
+      const auto most_right_lane = route_handler.getMostRightLanelet(lane, true, true);
+      const auto most_right_opposite_lanes =
+        route_handler.getRightOppositeLanelets(most_right_lane);
+      return most_right_opposite_lanes.empty() ? most_right_lane
+                                               : most_right_opposite_lanes.front();
+    } else {
+      const auto most_left_lane = route_handler.getMostLeftLanelet(lane, true, true);
```

Comment:
It used to be route_handler.getMostLeftLanelet(lane, false, true);, but now the booleans are true,true. Is there a reason?

Reply:
Since the avoidance module uses that lane, the goal planner must also check the same lane.